### DISCUSSION TITLE
Design of Starlight

### DISCUSSION THREAD

**Discussion by illright** [withastro/starlight#1843] [Feature Requests]
### What version of `starlight` are you using?

0.22.2

### What is your idea?

How is Starlight being designed? I like the minimalist style of the default theme, but there are certain aspects that I think could be improved, for example, the background color of asides.

I'd like to peek under the curtain of the design process and see if it's something I could get involved in, or if there is an established design process already. Is this an aspect that you accept contributions in?

### Why is this feature necessary?

Making Starlight look great :)

### Do you have examples of this feature in other projects?

I'm quite fond of the design of VitePress: https://vitepress.dev/

### Participation

- [X] I am willing to submit a pull request for this feature.

**Comment by delucis** [comment]
Hi @illright! The initial design phase for Starlight was led by @doodlemarks with an open review process in the `#starlight` channel in our Discord server (linking to the [kick-off the message](https://discord.com/channels/830184174198718474/1070481941863878697/1070679287537090590) in that thread for posterity). (For anyone who needs a Discord invite, everyone is welcome at <https://astro.build/chat>!)

The design was done in Figma and the final result of that initial design work can be viewed in [the Figma design document](https://www.figma.com/design/GrDkcguAR7tSWPFuLVDixq/Theme---Starlight-Docs?node-id=0%3A1&t=tA3qy9BNQCC0v3Tp-1), which is our main reference point for design questions.

Since then we have made ad-hoc small design decisions as we worked on details and new features. We’re definitely open to people getting involved in all aspects of the project, but bear in mind that we often have to weigh up how changes impact existing users, so try to resist changing styling for purely subjective and stylistic reasons, in favour of aiming to address usability and accessibility issues. Also we might sometimes choose a simpler/more minimal design because it can be a stable foundation for Starlight users to customize and modify, which is also an important value for us.

Hope that doesn’t sound too discouraging — we welcome input — but also want to set expectations respectfully if you’re looking to get involved!

**Comment by Fryuni** [comment]
Hey @illright, besides what Chris said, you might want to have a look at [this other discussion](https://github.com/withastro/starlight/discussions/1701) and give your opinions there as well since you are interested in the design side of the project 😄
